### PR TITLE
Deploy develop branch to staging environment in AWS

### PR DESCRIPTION
**Pull request by griggheo**
Type: **feature**

## Issue
This PR updates the CircleCI workflow to account for the latest version of the propel CLI tool. It deploys 2 services:
* sk-core (the Reaction core service) - version specified in propel.yaml
* sk-storefront (the reaction-next-starterkit service) - version is the latest one build in CircleCI off of the develop branch

## Testing
* perform functional testing on https://storefront.staging.reactioncommerce.com/

### CODE REVIEW

**Comment by griggheo** [comment]
Note that the deployment only pushes 1 service available at https://storefront.staging.reactioncommerce.com. The backend reaction-core service is available at https://reaction-core.staging.reactioncommerce.com.

File: propel.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  task-definition:
+    name: staging-storefront-core
+    containers:
+    - name: nginx
+      cpu: 128
+      memory: 256
+      port_mappings:
+      - container_port: 8082
+        host_port: 8082
+      image: sportsdirect/nginx
```
Comment (nnnnat) [inline propel.yaml]: @griggheo this image doesn't seem correct.

**Comment by willopez** [comment]
@griggheo can this PR be closed?

**Comment by griggheo** [comment]
Yes let's close it. At some point we'll have a new staging environment
dedicated to OSS reaction and then we can revisit.

**Pull request closed without merging**